src/api/app/components/bs_request_comment_component.html.haml

```diff
@@ -8,8 +8,39 @@
       wrote
       = link_to("#comment-#{comment.id}", title: l(comment.created_at.utc), name: "comment-#{comment.id}") do
         #{time_ago_in_words(comment.created_at)} ago
-    = helpers.render_as_markdown(comment)
-    = render partial: 'webui/comment/reply', locals: { comment: comment, level: 0, commentable: commentable }
+    .comment-bubble
+      - if policy(comment).update? || policy(comment).destroy?
+        .dropdown.dropleft.float-right
+          = link_to('#', role: 'button', 'data-toggle': 'dropdown', 'aria-expanded': 'false', class: 'text-dark') do
+            %i.fas.fa-ellipsis
+          .dropdown-menu
+            - if policy(comment).update?
+              = link_to("#edit_form_of_#{comment.id}", id: "edit_button_of_#{comment.id}", 'data-toggle': 'collapse',
+                        class: 'dropdown-item collapsed') do
+                Edit
+            - if policy(comment).destroy?
+              = link_to('#', data: { toggle: 'modal', target: "#delete-comment-modal-#{comment.id}" },
+                        class: 'dropdown-item delete_link', title: 'Delete comment') do
+                Delete
+
+      = helpers.render_as_markdown(comment)
+
+      - if User.session
+        = link_to("#reply_form_of_#{comment.id}", id: "reply_button_of_#{comment.id}", 'data-toggle': 'collapse',
+                  class: 'd-block mt-2 font-italic small collapsed') do
+          Reply
+        .collapse{ id: "reply_form_of_#{comment.id}" }
```

Comment:
💭  UX wise I would expect whatever I add here to be part of the box that you render the form is in...

Reply:
True, I'll render this form outside of the comment bubble.

Reply:
This is addressed.